index.d.ts

```diff
@@ -0,0 +1,128 @@
+declare module 'fluture' {
+  interface ResolveFunction<T> {
+    (value: T): void
+  }
+  interface RejectFunction<S> {
+    (error: S): void
+  }
+
+  interface FunctionReturning<T> {
+    (...t: any[]): T
+  }
+  abstract class Future<T, S> {
+    done<T, S> (fn: (err: S, value: T) => void): Future<T, S>
+
+    fork (reject: RejectFunction<S>, resolve: ResolveFunction<T>): void
+  }
```

Comment:
Fluture has many more methods than just `fork` and `done`. Almost every function in Fluture is also available as a method. We should provide typings for these as well.